docs/connector-development/config-based/overview.md

```diff
@@ -15,22 +15,78 @@ The process then submits HTTP requests to the API endpoint, and extracts records
 
 See the [connector definition section](yaml-structure.md) for more information on the YAML file describing the connector.
 
+## Is this API supported?
+
+Not all APIs are can be built using this framework because its featureset is still limited.
+This section describes guidelines for determining whether a connector for a given API can be built using the config-based framework.
+
+Refer to the API's documentation to answer the following questions:
+
+### Is this an HTTP REST API returning data as JSON?
+
+The API documentation should show which HTTP method must be used to retrieve data from the API.
+For example, the [documentation for the Exchange Rates Data API](https://apilayer.com/marketplace/exchangerates_data-api#documentation-tab) says the GET method should be used, and that the response is a JSON object.
+
+Other API types such as SOAP are not supported.
```

Comment:
tiniest nit but maybe mention another protocol like GraphQL or RPC isn't supported just for more examples

```suggestion
Other API types such as SOAP or GraphQL are not supported.
```